docs/cow-protocol/concepts/flash-loans/integrators.md

```diff
@@ -19,9 +19,13 @@ It is important to ensure that the flash loan gas overhead is added to the slipp
 }
 ````
 
-- **lender (optional):** the contract that could be used to borrow the funds from. For example `0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA` for Maker DAO, or `0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2` for AAVE.
-- **borrower (optional):** who should receive the borrowed tokens.
+- **lender (optional):** the contract that could be used to borrow the funds from. For example `0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA` for Maker DAO, or `0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2` for AAVE. If the value is not provided the solver will choose a sensible default.
```

Comment:
```suggestion
- **lender (optional):** the contract that could be used to borrow the funds from. For example `0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA` for any `ERC-3156` compliant lender contract, or `0x87870Bca3F3fD6335C3F4ce8392D69350B4fA4E2` for AAVE. If the value is not provided the solver will choose a sensible default.
```

Reply:
Also I believe we currently don't have logic to pick a sensible default. Perhaps we should mark this as non-optional in the docs.